### ISSUE TITLE
Search relative file in file’s directory, not current directory

### ISSUE THREAD

**Issue by hoelbezier** [kuznetsss/shswitch#1]
If `autchdir` is not set, current process directory might not be the same as the edited file directory, which can prevent shswitch from finding the relative file.

**Comment by kuznetsss** [comment]
That's true. Actually I don't use this plugin for a long time because clangd provides this functionality.
